bin/console

```diff
@@ -38,6 +42,22 @@ if ($_SERVER['APP_DEBUG']) {
     }
 }
 
-$kernel = new Kernel($_SERVER['APP_ENV'], (bool) $_SERVER['APP_DEBUG']);
+$isBlackfireProfilingRequested = $input->hasParameterOption('--blackfire', true);
```

Comment:
Suggestion: I'd suggest having an env variable o enable it Ex:

```
if ($_ENV['BLACKFIRE_ENABLED']) {
...
}

Reply:
Great idea! Added! :)